tools/interactive/interactivetool_anylabeling.xml

```diff
@@ -0,0 +1,83 @@
+<tool id="interactive_tool_anylabeling" tool_type="interactive" name="AnyLabeling Interactive" version="@VERSION@" profile="24.2">
+    <description>Effortless data labeling with AI support from YOLO and Segment Anything!</description>
+    <icon src="anylabeling.png"/>
+    <requirements>
+        <container type="docker">quay.io/bgruening/anylabeling:0.4.30</container>
+    </requirements>
+    <entry_points>
+        <entry_point name="AnyLabeling" requires_domain="True">
+            <port>5800</port>
+        </entry_point>
+    </entry_points>
+    <command detect_errors="exit_code">
+    <![CDATA[
+         export HOME=\$PWD &&
+         
+         mkdir -p ./home ./home/output ./home/input_images &&
+	 chown -R 1000:1000 ./home &&
+
+         #for $i,$img in enumerate($input_images):
+            ln -s '$img' './home/input_images/${img.element_identifier}' &&
+         #end for
+
+         #if $classes:
+             mkdir -p ./home/input_classes &&
+             #for $i,$cls in enumerate($classes):
+                 ln -s '$cls' './home/input_classes/${cls.element_identifier}' &&
+             #end for
+         #end if
+
+         #if $annotations:
+             mkdir -p ./home/input_annotations &&
+             #for $i,$ann in enumerate($annotations):
+                 ln -s '$ann' './home/input_annotations/${ann.element_identifier}' &&
+             #end for
+         #end if
+
+         #if $models:
+             #import glob,os
+             mkdir -p ./home/input_models &&
+	     tar -C ./home/input_models -xvf $models &&
+             #for $f in sorted(glob.glob("./home/input_models/*")):
+                 ln -s "$f" "./home/input_models/${os.path.basename($f)}" &&
+             #end for    
+         #end if 
+
+        /init 
+         
+
+    ]]>
+    </command>
+    <inputs>
+        <param name="input_images" type="data" format="jpg,png,tiff,bmp,gif,pcx,ppm,psd,pbm,pgm,eps" multiple="true"  label="Input images"/>
+        <param name="annotations" type="data" format="txt,json" optional="true" multiple="true" label="Pre-existing annotations"/>
+        <param name="classes" type="data" format="txt" optional="true" multiple="true" label="Classes file (one class per line)"/>
+        <param name="models" type="data" format="tar" optional="true" label="A tarball containing custom model files and yaml files"/>
```

Comment:
It would be helpful to mention maybe in the tool help where you get the models.

Reply:
added, together with changing the class input to single. could you do another round of review, @bgruening Thank you